collect.py

```diff
@@ -211,12 +212,12 @@ def run_test(bug, browser, driver, op_sequence=None):
     return saved_sequence
 
 
-def run_tests(firefox_driver, chrome_driver):
+def run_tests(firefox_driver, chrome_driver, bugs):
     set_timeouts(firefox_driver)
     set_timeouts(chrome_driver)
-
+    
```

Comment:
For example this is `./collect.py:218:1: W293 blank line contains whitespace`. The whitespaces should be removed here.